pom.xml

```diff
@@ -35,17 +33,30 @@
 			</plugin>
 		</plugins>
 	</build>
+	<dependencies>
+		<!-- Required for e-mail resolving (MailAddressResolver,...) -->
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>mailer</artifactId>
+			<version>1.8</version>
+		</dependency>
+	</dependencies>
     <developers>
         <developer>
             <id>imod</id>
             <name>Dominik Bartholdi</name>
             <email>-</email>
         </developer>
+		<developer>
+			<id>nicobo</id>
+			<name>Nicolas BONARDELLE</name>
+			<url>http://nicolabs.net/contact</url>
+		</developer>
     </developers>	
 	<scm>
-		<connection>scm:git:git://github.com/jenkins/script-realm-plugin.git</connection>
-		<developerConnection>scm:git:ssh://git@github.com/jenkinsci/script-realm-plugin.git</developerConnection>
-		<url>http://github.com/jenkins/script-realm-plugin</url>
+		<connection>scm:git:git://github.com/nicolabs/script-realm-plugin.git</connection>
+		<developerConnection>scm:git:ssh://git@github.com/nicolabs/script-realm-plugin.git</developerConnection>
+		<url>https://github.com/nicolabs/script-realm-plugin</url>
```

Comment:
don't change anything within the `scm` section - this will stop us from doing a proper release